src/createSliderWithTooltip.jsx

```diff
@@ -26,13 +28,25 @@ export default function createSliderWithTooltip(Component) {
       });
     }
     handleWithTooltip = ({ value, dragging, index, disabled, ...restProps }) => {
-      const { tipFormatter } = this.props;
+      const {
+        tipFormatter,
+        tipProps,
+      } = this.props;
+
+      const {
+        prefixCls = 'rc-slider-tooltip',
+        overlay = tipFormatter(value),
+        placement = 'top',
```

Comment:
@zeak 

The difference is maintainability. But i think right now is ok. I will merge this and release a minor version. Thanks.